setup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+import glob
+
+scripts = glob.glob("scripts/*")
+templates = glob.glob("templates/*")
+model_spec = glob.glob("model_spec/*")
+
+setup(name='acis_thermal_check',
+      packages=find_packages(),
+      version='1.0.0',
+      description='ACIS Thermal Model Validation Library',
+      author='John ZuHone',
+      author_email='jzuhone@gmail.com',
+      url='http://github.com/acisops/acis_thermal_check',
+      download_url='https://github.com/acisops/acis_thermal_check/tarball/1.0.0',
+      data_files=[('templates', templates),
+                  ('model_spec', model_spec)],
+      scripts=scripts,
+      classifiers=[
+          'Intended Audience :: Science/Research',
+          'Operating System :: OS Independent',
+          'Programming Language :: Python :: 2.7',
+          'Programming Language :: Python :: 3.4',
```

Comment:
Whoops, yeah. Sorry, I copied a `setup.py` over from another project as a template and forgot to take that out.